### PR TITLE
feat(ui): add initial project grid on work page

### PR DESCRIPTION
**Pull request by kjwessa**
TL;DR
This PR introduces significant enhancements and cleanups to the WorkGrid and its associated components.

What changed?
1. Deleted several unused image files from the `/public/images` directory.
2. Added new image files for project thumbnails to the `/public/images` directory.
3. Created the `WorkGrid` component to dynamically display project data using the `WorkCard` component.
4. Updated the `WorkCard` component to remove unused fields (`location` and `year`).
5. Updated project data JSON files to reference the new images in the `/public/images` directory.
6. Modified the main work page (`page.tsx`) to include the `WorkGrid` component.

How to test?
1. Run the application locally.
2. Verify that the main work page displays the projects with the correct data and images.
3. Ensure that there are no broken images or missing data in the project cards.

Why make this change?
To improve the maintainability of the project by cleaning up unused assets and to enhance the visual presentation of the project data on the work page.

---

### CODE REVIEW

**Comment by kjwessa** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/kjwessa/brewww-studio/56?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#56** <a href="https://app.graphite.dev/github/pr/kjwessa/brewww-studio/56?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#55** <a href="https://app.graphite.dev/github/pr/kjwessa/brewww-studio/55?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kjwessa and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by kjwessa** [comment]
### Merge activity

* **Jun 24, 11:45 AM EDT**: @kjwessa started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/kjwessa/brewww-studio/56).
